SWLOR.Game.Server/Feature/GuiDefinition/ViewModel/SettingsViewModel.cs

```diff
@@ -82,6 +92,9 @@ private void UpdateHolonetSetting()
         {
             SetLocalBool(Player, "DISPLAY_HOLONET", DisplayHolonetChannel);
         }
-
+        private void UpdateSubdualMode()
+        {
+            SetLocalBool(Player, "SUBDUAL_MODE", SubdualMode);
```

Comment:
I wouldn't bother with the local variable. The only reason we do it with the Holonet is because of how frequent it is and how many players are involved (would require an indeterminate number of DB calls). I would get rid of this and change your check in Death.cs to check the player DB flag instead.

Reply:
Done